README.md

```diff
@@ -1,75 +1,97 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
[REQUIRED] Kindly make your README more descriptive. Use the [Microverse template](https://github.com/microverseinc/readme-template)